app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,22 @@
+class OrdersController < ApplicationController
+
+  def index
+    @orders = current_user.orders
+  end
+
+  def create
+    order = current_user.orders.create(total_price: @cart.total)
+    submit_order(order)
+    session.delete(:cart)
+    flash[:success] = "Mischief is coming your way, you have successfully placed your order!"
+    redirect_to orders_path
+  end
+
+  def submit_order(order)
```

Comment:
This would really be the only method in a Checkout PORO, just not sure if having this method in the controller makes it not a skinny controller. @Carmer @s-espinosa any thoughts?